app/server/pom.xml

```
@@ -24,6 +24,7 @@
         <project.version>1.0-SNAPSHOT</project.version>
         <!-- By default skip the dockerization step. Only activate if necessary -->
         <skipDockerBuild>true</skipDockerBuild>
+        <log4j2.version>2.16.0</log4j2.version>
```

Should we do 2.17 directly?

```suggestion
        <log4j2.version>2.17.0</log4j2.version>
```